README.md

```diff
@@ -37,9 +36,9 @@ Percona is dedicated to **keeping open source open**. Whenever possible, we stri
 
 ## How to get involved
 
-We encourage contributions and are always looking for new members that are as dedicated to serving the community as we are. The [Contributing Guide](https://github.com/percona/percona-xtradb-cluster/blob/8.0/doc/source/contributing.md) contains how you can contribute.
+We encourage contributions and are always looking for new members that are as dedicated to serving the community as we are. The [Contributing Guide](https://github.com/percona/pxc-docs/blob/8.0/contributing.md) contains how you can contribute.
```

Comment:
instead of "contains how can contribute" maybe "describes how you can contribute"?

Reply:
Or "contains instructions on how you can contribute"

Reply:
Updated